extensions/wikia/WikiaMobile/WikiaMobileCategoryService.class.php

```diff
@@ -80,6 +80,7 @@ public function alphabeticalList() {
 			$title = $categoryPage->getTitle();
 			$category = Category::newFromTitle( $title );
 			$collections = $this->model->getCollection( $category, $format );
+			ksort( $collections['items'] );
```

Comment:
OK, after some digging I know the thing is more complex… https://www.mediawiki.org/wiki/Manual:$wgCategoryCollation